### ISSUE TITLE
the performance of the u-t?  

### ISSUE THREAD

**Issue by wanghangege** [HXLH50K/U-Net-Transformer#5]
Have you ever done some test or on the dataset of official paper?

**Comment by HXLH50K** [comment]
No.  The performance on my own dataset is poor, I think there are some problems with the code, and I am too lazy to fix
╮（╯＿╰）╭
